docs/dg/dev/backend-development/data-manipulation/queue/queue.md

```diff
@@ -129,6 +129,25 @@ class SampleQueueMessageProcessorPlugin implements QueueMessageProcessorPluginIn
 ?>
 ```
 
+### Configuration for chunk size
+
+Instead of relying on the `getChunkSize()` method in the plugin, you can use a project-level configuration to define a more flexible chunk size for any queue. Here's how you can configure it using`QueueConstants::QUEUE_MESSAGE_CHUNK_SIZE_MAP`:
+
+```php
+<?php
+
+use Spryker\Shared\Queue\QueueConstants;
+
+$config[QueueConstants::QUEUE_MESSAGE_CHUNK_SIZE_MAP] = [
+    'publish' => 2000,
+    'event' => 1000,
+    'sync.search.product' => 100
+];
+?>
+```
+
+This configuration takes precedence over the `getChunkSize()` method. If a chunk size is defined for a queue in `QUEUE_MESSAGE_CHUNK_SIZE_MAP`, that size is used. Otherwise, the chunk size falls back to the value returned by the `getChunkSize()` method of the `QueueMessageProcessorPluginInterface`.
+
 Register the plugin in `QueueDependencyProvider::getProcessorMessagePlugins()`:
```

Comment:
How is this related to this section?

Reply:
Processor plugins have some gaps related to chunk size configuration, so the new configuration is more prioritized/preferable than the default one.